index.js

```diff
@@ -17,9 +17,10 @@ const core = require('./core.js');
 const mkdirp = require('mkdirp');
 const replHistory = require('repl.history');
 const minimist = require('minimist');
+const ip = require('ip');
+const {version} = require('./package.json');
```

Comment:
Nit, but I would leave `package.json` below. It's not a module to be included, so probably doesn't belong with those.